ansible_role/defaults/main.yml

```diff
@@ -164,3 +164,5 @@ quay_registry:
 
 # conditionally set later based on Cluster configuration
 default_registries: []
+
+tests_deploy_broker: true
```

Comment:
Looking at the failure from travis. I think it's more correct to take this out of this file and add it to the vars of the ansible_role/playbooks/test.yml

Reply:
Understood. I'll do so in a few and see if that gets it cleaned up.